schema.graphql

```diff
@@ -825,10 +832,16 @@ type NameSignal @entity {
   # unrealized gains for their current balance, based on the time since the last exit/entry of the curve
   "Summation of realized rewards from before the last time the curator entered the curation curve"
   realizedRewards: BigInt!
+
   "Curator average cost basis for this name signal on this subgraph"
-  averageCostBasis: BigDecimal! # note this is ONLY name signal. This is okay for the protocol for now
-  "averageCostBasis / nameSignal"
-  averageCostBasisPerSignal: BigDecimal!
+  nameSignalAverageCostBasis: BigDecimal! # note this is ONLY name signal. This is okay for the protocol for now
+  "nameSignalAverageCostBasis / nameSignal"
+  nameSignalAverageCostBasisPerSignal: BigDecimal!
+
+  "Curator average cost basis for the version signal on this subgraph name pool"
+  signalAverageCostBasis: BigDecimal!
```

Comment:
this would be effected by the comment above, where i mention that `namesignal.signal` is dynamic.

Reply:
Somewhat related to the answer above.

Reply:
Same as above, should be fixed now